pyg_lib/csrc/sampler/cpu/indices_tracker.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <ATen/ATen.h>
+
+#include "parallel_hashmap/phmap.h"
+
+namespace pyg {
+namespace sampler {
+
+template <typename scalar_t>
+class IndicesTracker {
```

Comment:
How is this different from `pyg::sampler::Mapper`? Can we unify the logic here?

Reply:
Main diff is set vs map usage (flat_hash set vs map). We can think how to unify it.